cli/mender-artifact/main.go

```diff
@@ -123,6 +110,22 @@ func getCliContext() *cli.App {
 		Usage: "The group(s) the artifact depends on",
 	}
 
+	//
+	// Common Payload flags
```

Comment:
Question: In general, I like the split of these, but should we really start naming them payload-depends? I liked Artifact Depends betterer

Reply:
The internal field in the spec is called `artifact_depends`, so you're right in that sense. However, it is per-payload, and inside `type-info`, and hence different from the top-level `artifact_depends`. I just wanted to make that distinction clear. I'm fine with renaming it, but I would like to keep global and per-payload flags clearly visible. Do you have any good name suggestions?

Reply:
Since I've now spent 5 minutes not coming up with anything better, I guess this is more than good enough :) In the `mender-artifact` repo these are called `type-info-depends` but that's worse imo.

Reply:
I agree, that's worse!